guides/common/modules/proc_installing-the-scc-manager.adoc

```diff
@@ -1,14 +1,33 @@
 [id="Installing_the_SCC_Manager_Plugin_{context}"]
 = Installing the SCC Manager Plug-in
 
-Perform the following step to install the _SCC Manager_ plug-in on your {Project}.
+Perform the following steps to install the _SCC Manager_ plug-in on your {Project}.
```

Comment:
IMHO something went wrong during rebasing to master. Would you mind having a look at this again? Thanks!

Reply:
I rebased it. I hope, there is no more issue :crossed_fingers:

Reply:
Fixed :+1: